presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -313,6 +313,12 @@ class SystemConfig : public ConfigBase {
 
   static constexpr std::string_view kAsyncDataCacheEnabled{
       "async-data-cache-enabled"};
+  /// If true, SSD cache is enabled by default and is disabled only if
+  /// `node_selection_strategy` is present and set to `NO_PREFERENCE`.
+  /// Otherwise, SSD cache is disabled by default and is enabled if
+  /// `node_selection_strategy` is present and set to `SOFT_AFFINITY`.
```

Comment:
I would leave out the implementation logic and only give high level introduction 
```
/// Specifies the default cache behavior, per query cache behavior will be based on both this config and session property `node_selection_strategy`
```

Reply:
I feel "per query cache behavior" is not that intuitive here and worth being called out?